### PR TITLE
Update Android test sheet

### LINKED ISSUE #121: Add settings test for Android

### PR DESCRIPTION
**Pull request by srirambv**
fixes #121

### CODE REVIEW

**Comment by srirambv** [comment]
Adding @SergeyZhukovsky for review

File: wikitemplate-android.md [hunk 1]
```diff
@@ -6,22 +6,25 @@
 
 ## Visual look
 
-- [ ] Make sure there is no words of Chromium after any merge
+- [ ] Make sure there after any merge
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: Make sure that after very merge:

File: wikitemplate-android.md [hunk 2]
```diff
@@ -6,22 +6,25 @@
...
+  - [ ] No Chromium words appear on normal or private tabs
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: Both Chromium/Chrome words are not being used in normal or private tabs

File: wikitemplate-android.md [hunk 3]
```diff
@@ -6,22 +6,25 @@
...
+  - [ ] No Chromium icons are shown in normal or private tabs
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: No Chromium/Chrome icons are being used in normal or private tabs

File: wikitemplate-android.md [hunk 4]
```diff
@@ -43,32 +46,37 @@ Pre-Requisite: Put previous build shortcut on home screen. Also have several sit
 ## Find in page
 
 - [ ] Ensure search box is shown when selected via the hamburger menu
-- [ ] Test successful find
-- [ ] Test forward and backward find navigation
-- [ ] Test failed find shows 0 results
+- [ ] Verify successful find
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: Verify that you can successfully find a word on the page

File: wikitemplate-android.md [hunk 5]
```diff
@@ -43,32 +46,37 @@ Pre-Requisite: Put previous build shortcut on home screen. Also have several sit
...
+- [ ] Verify forward and backward find navigation
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: Verify that the forward and backward navigation icons under find are working

File: wikitemplate-android.md [hunk 6]
```diff
@@ -43,32 +46,37 @@ Pre-Requisite: Put previous build shortcut on home screen. Also have several sit
...
+- [ ] Verify failed find shows 0 results
 
 ## Site hacks
 
 - [ ] Verify https://www.twitch.tv/adobe sub-page loads a video and you can play it
 
+## Settings and Bottom bar
+
+- [ ] Verify changing default settings are retained and doesn't cause the browser to crash
+- [ ] Verify bottom bar buttons(Home/Bookmark/Search/Tabs) work as expected
```
Comment (kjozwiak) [inline wikitemplate-android.md]: nit: space between "buttons(Home/Bookmark/Search/Tabs)"

**Pull request merged**